src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/gateways/ReportedAdjudication.kt

```diff
@@ -22,7 +22,8 @@ data class ReportedAdjudication(
       dateTimeReportExpires = dateTimeReportExpires,
```

Comment:
We need to remove this as it is now a duplicate - given that we provide the handover deadline with the incident details